maquetado/signup.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Sign Up</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+    <form class="info-container column">
+      <img src ="./assets/wolox.png" class="login-img" alt="Wolox">
+      <h3 class="input-description">Nombre</h3>
+      <input name="name" id="name" class="input-field">
+      <h3 class="input-description">Apellido</h3>
+      <input name="surename" id="surename" class="input-field">
+      <h3 class="input-description">Email</h3>
+      <input name="email" id="email" class="input-field">
+      <h3 class="input-description">Password</h3>
+      <input name="pwd" id="password" class="input-field">
+      <h3 class="input-description">Confirmación de Password</h3>
+      <input name="pwdconfirmation" id="pwdconfirmation" class="input-field">
+      <button href="tbdef" class="confirmation-detail">Sign Up</button>
```

Comment:
Same as in the other PR. This shouldn't use `href` attribute since this is only meant for `a` tags. Also, you should add `type="submit"` to your button.